### DISCUSSION TITLE
Wrap text in boolean labels

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13964] [Feature Requests]
### Describe the Bug

Cannot see the whole text of the option.
![Screenshot from 2022-06-17 11-28-50](https://user-images.githubusercontent.com/95102939/174281058-d9bd868a-2e09-4275-818d-6b41a9114fd8.png)

So, how about wrapping text or adding a title tag so hovering over it will show the complete text?

### To Reproduce

For all M2Ms?

### What version of Directus are you using?

9.12.2

### What version of Node.js are you using?

12.22.5

### What database are you using?

mariadb

### What browser are you using?

firefox

### How are you deploying Directus?

local

**Comment by sxsws** [comment]
Which would be best - to wrap the text, or add a title tag to the element? I'm thinking the first would be the proper way to solve the problem, whilst the latter would be easier but... not as ui friendly.

**Comment by benhaynes** [comment]
Adding a tooltip (ideally only if truncated) is the preferred solution.

**Comment by sxsws** [comment]
But that would involve the user needing to move the cursor precisely over the text and waiting to see part of the form. Wrapping puts the text infront of the user immediately, though that will mean breaking the nice layout in its present form, but a div clear should solve that?

**Comment by benhaynes** [comment]
It's a tradeoff, we try to make the form have UX/UI that is hard to break just by adding extra long text... which is why we typically don't wrap things like this. The options are would be to shorten the text, make this toggle full-width, use an interface with more space (or build a custom one), or have the tooltips as a safeguard.

**Reply by koksikus** [reply]
It seems to me that the best solution is a popup window that would show all the content. This could be applied to every field and view - possibly an option to turn it off in the settings. When viewing the table, it would solve the problem that you have to click on each record to view the content or to make the photo larger. And so, after hovering the cursor over the field, you can preview the content.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
